### DISCUSSION TITLE
how to create a filled polygon with given set of vertices and add it to already existing scene?

### DISCUSSION THREAD

**Discussion by bramanat94** [foxglove/community#1017] [Q&A]
how to create a filled polygon with given set of vertices and add it to already existing scene?

**Comment by nealwang123** [comment]
no body answer?

**Comment by jtbandes** [comment]
There is no "filled polygon" concept built in but there is a [TriangleListPrimitive](https://docs.foxglove.dev/docs/sdk/schemas/triangle-list-primitive) which can be used if you convert the polygon into a set of triangles.
